src/components/Api.js

```diff
@@ -0,0 +1,212 @@
+import Section from './Section.js';
+import {createNewCard} from '../utils/utils.js';
+import {cardContainerClass} from '../utils/constants.js';
+import { userInfo, formValidatorCardModal } from '../pages/index.js';
+
+export default class Api{
+    constructor(){
+      this._baseUrl = 'https://around-api.en.tripleten-services.com/v1',
+      this._headers = {
+        authorization: "4c45d989-e1aa-4bb6-a467-6ac9c46f3dac",
+        "Content-Type": "application/json"
+      };
+      this._headers1 = {
+        authorization: "4c45d989-e1aa-4bb6-a467-6ac9c46f3dac",
+      }
+    }
+
+    loadUserInfo(){
+      return fetch(`${this._baseUrl}/users/me`, {
+        headers: this._headers1
+      })
+      .then(res => {
+        if(res.ok){
+          return res.json();
+        } else{
+          return Promise.reject(`Error: ${res.status}`);
+        }
+      })
+      .then(data => {
+        userInfo.setUserInfo(data);
+        document.querySelector('.profile__image').src = data.avatar;
+      })
+      .catch(err => {
+        console.error(err);
```

Comment:
It’s better not to use `catch` in `Api` because you need to `catch` possible errors at the very **end** of any request and this end is in `index.js`  after all blocks `then`. And in real projects blocks `catch` usually open error popups and so on.

Please, remove it from `Api` and add it in `index.js`.